examples/pytorch/pointcloud/pointnet/pointnet_cls.py

```diff
@@ -0,0 +1,145 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.autograd import Variable
+import numpy as np
+
+class PointNetCls(nn.Module):
+    def __init__(self, output_classes, input_dims=3,
+                 dropout_prob=0.5, use_transform=True):
+        super(PointNetCls, self).__init__()
```

Comment:
I would suggest using self.conv1_dim instead of 64 here. When we switch to another dataset, we may need to change the dim and may avoid manually change every line of code. As well as for other layers.